src/components/Navbar/NavLinks.jsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react';
+import { NavLink, useLocation, useNavigate } from 'react-router-dom';
+import { destroySession, TOKENKEY } from 'util/auth';
+
+// eslint-disable-next-line react/prop-types
+const NavLinks = ({ toggleNavbar }) => {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  // const [token, setToken] = useState("");
```

Comment:
Remove commented out code., it is better remove, can always reference in the git history